### DISCUSSION TITLE
Global config: look to /etc/lsd.yaml

### DISCUSSION THREAD

**Discussion by jehon** [lsd-rs/lsd#867] [Ideas]
Do you know if lsd would look to a global config file? I am thinking about a /etc/lsd.yaml.

I didn't find anything in the doc (and had a glace at the code), but I didn't find anything...

Would that be a good idea?

PS: I could perhaps make a pr if this is a good idea

**Comment by Duz-Dev** [comment]
Hello, I also have the same question. I would like to know if there is a global configuration file as well. Sorry if my English is bad, I use ChatGPT to translate. lol

**Comment by zwpaper** [comment]
Sounds reasonable, maybe we can introduce a global configuration dir, and look for /etc/lsd/*.yaml
